pkg/api/metrics.go

```diff
@@ -81,6 +81,7 @@ func GetTestDataScenarios(c *m.ReqContext) Response {
 // Generates a index out of range error
 func GenerateError(c *m.ReqContext) Response {
 	var array []string
+	// nolint: govet
```

Comment:
I guess this can be replaced with raising a panic

```go
panic("panic: runtime error: index out of range")
```

Reply:
@bergquist What is the goal of `GenerateError` itself?